### PR TITLE
Document `bool_and`, `bool_or`, `jsonb_agg`, `jsonb_object_agg`

### LINKED ISSUE #918: Document `bool_and`, `bool_or`, `jsonb_agg`, `jsonb_object_agg`
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/9452

### Which part(s) of the docs might be affected or should be updated? And how?

https://www.risingwave.dev/docs/upcoming/sql-function-aggregate/

### Reference

_No response_

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info
- **Description**: 
[ What's changed? ]

- **Preview**: 
[ Paste the preview link to the edited page here. Edit this item after you submit the PR and the preview is ready. ]

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/9452

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/918

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/sql/functions-operators/sql-function-aggregate.md [hunk 1]
```diff
@@ -4,17 +4,136 @@ slug: /sql-function-aggregate
...
+count ( expression ) -> bigint    
+```  
+
+Input types include bool, smallint, int, bigint, numeric, real, double precision, and string.  
+
+---  
+
+### `jsonb_agg`
+
+Aggregates values, including nulls, as a JSON array.
```
Comment (wangrunji0408) [inline docs/sql/functions-operators/sql-function-aggregate.md]: It's worth mention that like `array_agg`, an optional `ORDER BY` can be specified to determine the order of the elements in the result array.

File: docs/sql/functions-operators/sql-function-aggregate.md [hunk 2]
```diff
@@ -4,17 +4,136 @@ slug: /sql-function-aggregate
...
+
+```sql title=Syntax  
+jsonb_agg ( expression ) -> jsonb    
+```
+
+Input types include boolean, int2/4/8, float4/8, varchar.
```
Comment (wangrunji0408) [inline docs/sql/functions-operators/sql-function-aggregate.md]:
```suggestion
Input types include boolean, smallint, int, bigint, real, double precision, varchar and jsonb.
```
P.S. the input should accept any types, but currently we only support the above types.

File: docs/sql/functions-operators/sql-function-aggregate.md [hunk 3]
```diff
@@ -4,17 +4,136 @@ slug: /sql-function-aggregate
...
+
+Aggregates name/value pairs as a JSON object.
+
+```sql title=Syntax  
+jsonb_object_agg ( key , value ) -> jsonb   
+```
+
+`key`: varchar only.
+
+`value`: boolean, int2/4/8, float4/8, varchar, null.
```
Comment (wangrunji0408) [inline docs/sql/functions-operators/sql-function-aggregate.md]:
```suggestion
`value`: null, boolean, smallint, int, bigint, real, double precision, varchar and jsonb.
```

File: docs/sql/functions-operators/sql-function-aggregate.md [hunk 4]
```diff
@@ -4,17 +4,136 @@ slug: /sql-function-aggregate
...
+min ( expression ) -> same as input type  
+```  
+
+Input types include smallint, int, bigint, numeric, real, double precision, and string.  
+
+---  
+
+### `string_agg`
+
+Combines non-null values into a string, separated by `delimiter_string`.
```
Comment (wangrunji0408) [inline docs/sql/functions-operators/sql-function-aggregate.md]: It also supports `ORDER BY` like `array_agg`.

**Review by emile-00 — APPROVED** [review]

**Pull request merged**